app/controllers/api/v3/courses_controller.rb

```diff
@@ -7,7 +7,7 @@ class CoursesController < API::V3::ApplicationController
 
       def index
         course_search = CourseSearchService.call(filter: params[:filter], sort: params[:sort], course_scope: @courses)
-
+        
```

Comment:
:eyes:

Reply:
Sorted

Reply:
Not quite sorted, should this be one or two lines in betwee?